receiver/collectdreceiver/collectd.go

```diff
@@ -22,6 +22,8 @@ import (
 
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
 	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/sanitize"
```

Comment:
Maybe file an issue: investigate if somebody (maybe golang official lib) exists for sanitization after all these big incident :)